### PR TITLE
Add RBG and Jim Weirich to names generator

### PR DESCRIPTION
**Pull request by cpuguy83**
Closes #41200

### CODE REVIEW

**Review by tianon — APPROVED** [review]
LGTM

**Review by thaJeztah — APPROVED** [review]
LGTM 🏄‍♀️

File: pkg/namesgenerator/names-generator.go [hunk 1]
```diff
@@ -374,6 +374,9 @@ var (
 		// Marie-Sophie Germain - French mathematician, physicist and philosopher. Known for her work on elasticity theory, number theory and philosophy. https://en.wikipedia.org/wiki/Sophie_Germain
 		"germain",
 
+		// Ruth Bader Ginsburg served as associate justice of the Supreme Court of the United States and spent much of her career as an advocate for gender equality and women's rights. https://en.wikipedia.org/wiki/Ruth_Bader_Ginsburg
```
Comment (AkihiroSuda) [inline pkg/namesgenerator/names-generator.go:377]: Does she belong to `notable scientists and hackers`?

**Comment by SvenDowideit** [comment]
Lgtm

File: pkg/namesgenerator/names-generator.go (hunk 1 above)
Comment (thaJeztah) [reply pkg/namesgenerator/names-generator.go:377]: I would say she definitely "hacked the system" 😇

**Comment by tianon** [comment]
Thank you so much for taking the time to make this contribution!  We're very sorry for how long it has taken us to come to a decision on it, but as of https://github.com/moby/moby/pull/43210 the `namesgenerator` package is now officially "frozen" against new additions.  There is some explanation there as to why it is frozen (and why it has taken us so long to come to a decision on PRs like yours 🙈), but https://github.com/moby/moby/pull/43210#issuecomment-1029934277 is especially good for a longer explanation. :heart:

**Pull request closed without merging**